scripts/event.js

```diff
@@ -27,6 +27,7 @@ elements.events.appendChild(elements.actions);
 elements.addButton = document.createElement("input");
 elements.addButton.type = "button";
 elements.addButton.value = "Add";
+elements.addButton.id = "button";
```

Comment:
Bit weird to have a selector like `button#button`

```suggestion
elements.addButton.classList.add("btn-info") // or other class
```